manifest.json

```diff
@@ -1,13 +1,23 @@
 {
+  "evmconnect": {
+    "image": "ghcr.io/hyperledger/firefly-evmconnect",
+    "tag": "v0.1.0-20220810-2",
+    "sha": "bf580fd1698d9d2665a134d394cdb28e549b9e573d5f11b51dbe117ac1683ef9"
+  },
```

Comment:
Looks like we ended up with duplicate entries here. Probably due to merging various PRs together and the fact that I also added this in another branch. Sorry.